pytorch_lightning/loggers/base.py

```diff
@@ -410,6 +420,10 @@ def name(self) -> str:
     def version(self) -> str:
         return '_'.join([str(logger.version) for logger in self._logger_iterable])
 
+    @property
+    def state_dict(self) -> Dict[Type, dict]:
+        return {type(logger): logger.state_dict for logger in self._logger_iterable if logger.state_dict}
+
```

Comment:
what if we have multiple loggers of the same type in the collection? this runs into the same issue as #6467